frontend/package.json

```diff
@@ -38,21 +38,21 @@
     "build:dev": "sh -ac '. ./.env.upstream; npm run lint && npm run build:kiali'",
     "build:kiali": "npm run i18n && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build --profile",
     "build:prod": "sh -ac '. ./.env.downstream; npm run build:kiali'",
-    "cypress": "cypress open -e TAGS=\"not @multi-cluster and not @ambient\"",
-    "cypress:ambient": "cypress open -e TAGS=\"@ambient\" && cypress open -e TAGS=\"@waypoint\" && cypress open -e TAGS=\"@waypoint-tracing\"",
+    "cypress": "cypress open -e TAGS=\"not @multi-cluster and not @ambient and not @waypoint and (not @waypoint-tracing or @tracing)\"",
+    "cypress:ambient": "cypress open -e TAGS=\"@ambient or @waypoint or @waypoint-tracing\"",
```

Comment:
I think It would make more sense to open different ones, like: 
"cypress:ambient": "cypress open -e TAGS=\"@ambient\"",
"cypress:waypoint": "cypress open -e TAGS=\"@waypoint or @waypoint-tracing\"",

Because of the environments, that are different

Reply:
It is fine as it is now, because they are run in that specific order, so I think that change is enough